### ISSUE TITLE
webp support for PHP 5.5

### ISSUE THREAD

**Issue by Haehnchen** [avalanche123/Imagine#271]
Any plans to implement google webp as supported file extension? its available since PHP 5.5
http://www.php.net/manual/en/function.imagewebp.php

simply adding is working, but there is more todo...

``` php
$formats = array('gif', 'jpeg', 'png', 'wbmp', 'xbm', 'webp');
```

**Comment by gsouf** [comment]
Any news on the subject ?

**Comment by gsouf** [comment]
As a fallback it's possible to use cwebp command.

Trying to check if the imagine adapter supports webp. Any way that is lighweight and common to all ImagineInterface implementation to check that it supports webp at the runtime ?

**Comment by tyler-g** [comment]
Also interested in this..  @gsouf it's possible to do that but still need full support in order to do image transformations.  Would be curious to know if anyone is doing this right now!

**Comment by gsouf** [comment]
@tyler-g  my example processes images transformations and saves them to png, then ``cwebp`` command transforms the png to webp

**Comment by tyler-g** [comment]
@gsouf trying to figure out how to do resizing and then write to webp, is that possible?

**Comment by GromNaN** [comment]
@avalanche123 Would you accept a PR that add WebP support ?

**Comment by mlocati** [comment]
Implemented in #504
